services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -720,6 +721,7 @@ export const useCurrentlyExistsAtGroup = (application: string): EnvironmentGroup
                     environments: envs,
                     distanceToUpstream: group.distanceToUpstream,
                     numberOfEnvsInGroup: group.environments.length,
+                    priority: group.priority,
```

Comment:
please provide screenshots and a little more details in the PR description

Reply:
Done, is it detailed enough?